toolchain/semantics/semantics_node.h

```diff
@@ -68,6 +68,21 @@ struct SemanticsCrossReferenceIRId : public IndexBase {
   }
 };
 
+// The ID of a boolean value.
+struct SemanticsBoolLiteralId : public IndexBase {
+  using IndexBase::IndexBase;
+  auto Print(llvm::raw_ostream& out) const -> void {
+    if (index == 0) {
+      out << "false";
+    } else if (index == 1) {
+      out << "true";
+    } else {
+      out << "int";
+      IndexBase::Print(out);
```

Comment:
```suggestion
      CARBON_FATAL() << "Invalid bool value: " << index;
```

Definitely shouldn't keep the `int` label, would be `bool`, but that seems only necessary if you create with Invalid or other indices. That seems to be unsupported behavior, so maybe nix it?